packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js

```diff
@@ -75,46 +65,12 @@ async function configure(context) {
   if (context.exeInfo.template.Resources.CloudFrontDistribution) {
     const { DistributionConfig } = context.exeInfo.template.Resources.CloudFrontDistribution.Properties;
 
-    const questions = [
-      {
-        name: 'DefaultRootObject',
-        type: 'input',
-        message: 'default object to return from origin',
-        default: DistributionConfig.DefaultRootObject,
-      },
-      {
-        name: 'DefaultCacheDefaultTTL',
-        type: 'input',
-        message: 'Default TTL for the default cache behavior',
-        default: DistributionConfig.DefaultCacheBehavior.DefaultTTL,
-      },
-      {
-        name: 'DefaultCacheMaxTTL',
-        type: 'input',
-        message: 'Max TTL for the default cache behavior',
-        default: DistributionConfig.DefaultCacheBehavior.MaxTTL,
-      },
-      {
-        name: 'DefaultCacheMinTTL',
-        type: 'input',
-        message: 'Min TTL for the default cache behavior',
-        default: DistributionConfig.DefaultCacheBehavior.MinTTL,
-      },
-      {
-        name: 'ConfigCustomError',
-        type: 'confirm',
-        message: 'Configure Custom Error Responses',
-        default: true,
-      },
-    ];
-
-    const answers = await inquirer.prompt(questions);
-    DistributionConfig.DefaultRootObject = answers.DefaultRootObject;
-    DistributionConfig.DefaultCacheBehavior.DefaultTTL = answers.DefaultCacheDefaultTTL;
-    DistributionConfig.DefaultCacheBehavior.MaxTTL = answers.DefaultCacheMaxTTL;
-    DistributionConfig.DefaultCacheBehavior.MinTTL = answers.DefaultCacheMinTTL;
+    DistributionConfig.DefaultRootObject = await prompter.input('default object to return from origin', { initial: DistributionConfig.DefaultRootObject });
```

Comment:
nit: lowercase d here for default where the other messages have a uppercase D

Reply:
agree that isn't standard, but for the scope of this PR will leave as is as this it's how it's been in the previous version: https://github.com/aws-amplify/amplify-cli/pull/11679/files#diff-09e35a12cba2a09eb33793e5438207fb56462e5359e9a4042f5fd8c513cf41ccL82, a change here will probably break some prompter tests